CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 - The MSRV is now 1.56.1
   ([#1792](https://github.com/nix-rust/nix/pull/1792))
+- Passes `addr` to `sys::mmap` as `Option<NonZeroUsize>` from `*mut c_void`.
```

Comment:
I find this wording confusing.  How about this?
```suggestion
- The `addr` argument to `sys::mman::mmap` is now of type `Option<NonZeroUsize>`.
```

Reply:
Updated.